profilers/perfProfiler.py

```diff
@@ -102,13 +102,20 @@ def get_stats_dir(self, dir: Path) -> Dict[str, PerfData]:
                 data_dict[binary.split(".")[0]] = data
         return data_dict
 
+    def update_capabilities_dir(self, dir: Path):
+        for binary in os.listdir(dir):
+            pth = dir.joinpath(binary)
+            execute_line = ["sudo", "setcap", "cap_sys_admin=ep", pth]
+            subprocess.run(execute_line)
```

Comment:
If user is `root` and `sudo` isn't installed there will be an uncaught error
Also, It would be a good idea to tell the user why `sudo` is needed here (why a password is needed)

Reply:
About first, will be it good to fix this way?
```python
execute_line = ["setcap", "cap_sys_admin=ep", pth]
try:
    subprocess.run(execute_line)
except some_exception:
    subprocess.run(["sudo"] + execute_line)
```
---
About the second one, I understand you

Reply:
It didn't work out that way(